exercises/concept/secrets/.meta/exemplar.py

```diff
@@ -0,0 +1,14 @@
+def secret_add(secret):

```

Comment:
Docstrings should be added for all functions here.  Please see [this exemplar.py file](https://github.com/exercism/python/blob/main/exercises/concept/meltdown-mitigation/.meta/exemplar.py) for an example of the format.